### PR TITLE
add and delete an element in queue 

### PR DESCRIPTION
**Pull request by YuweAEC**
image

the add delete are previous works
code for list the items and make new case and call it into main function is todays work

### CODE REVIEW

File: daily-code/queue.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#include<stdio.h>
+#define MAX 5 //size of the queue
+
+void add(int x);
+void deletion(); 
+int a[MAX],front,rare;  //global declaration of queue
```
Comment (gauravsinghaec) [inline daily-code/queue.c]: Change the name, like queue[MAX]

File: daily-code/queue.c [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		front++;
+}
+
+void main()	
+{
+	int ch,x;  //ch is choice
+	front=rare=-1;  //initially queue is empty
+	do
+	{
+		printf("\n1. Add\n2. Delete\n3. List\n4. Exit\nEnter your choice: ");
```
Comment (gauravsinghaec) [inline daily-code/queue.c:42]: Good

File: daily-code/queue.c [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		scanf("%d",&ch);
+		switch(ch)
+		{
+			case 1: printf("Enter the element to be added: "); 		
+					scanf("%d",&x);
+					add(x); //call add function
+					break;
+			case 2: deletion();  //call delete function
```
Comment (gauravsinghaec) [inline daily-code/queue.c]: the correct name is delete ()

File: daily-code/queue.c [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	printf("the deleted element is %d",a[front]);  
+	if(front==rare)   //	check if only one element is present in queue
+		front=rare=-1; 
+	else
+		front++;
+}
+
+void main()	
+{
+	int ch,x;  //ch is choice
```
Comment (gauravsinghaec) [inline daily-code/queue.c]: right as choice, no need to use short variable name

File: daily-code/queue.c (hunk 1 above)
Comment (gauravsinghaec) [inline daily-code/queue.c]: it will be `rear` not rare

Comment (YuweAEC) [reply daily-code/queue.c]: Okay

**Review by gauravsinghaec — APPROVED** [review]

**Pull request merged**